### PR TITLE
Don't request an empty paint point

### PR DESCRIPTION
**Pull request by bvaughn**
This was a silly oversight.

### CODE REVIEW

File: packages/protocol/StreamingScreenShotCache.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const StreamingScreenShotCache = createStreamingCache<
 
     const paintPoint = findMostRecentPaint(time);
 
-    if (paintPoint) {
+    if (paintPoint && paintPoint.paintHash) {
```
Comment (jcmorrow) [inline packages/protocol/StreamingScreenShotCache.ts:46]: Out of curiosity, do you usually opt for this syntax over `paintPoint?.paintHash`?

**Review by jcmorrow — APPROVED** [review]

File: packages/protocol/StreamingScreenShotCache.ts (hunk 1 above)
Comment (bvaughn) [reply packages/protocol/StreamingScreenShotCache.ts:46]: It's subjective, but I think it's more explicit that we're accounting for two cases: no paint found and an empty paint

**Review by markerikson — APPROVED** [review]

**Comment by bvaughn** [comment]
Not sure what's going on with the backend. Lots of e2e test failures because of session creation failing.

**Comment by jcmorrow** [comment]
> **Edit** Seems like a spike in `processRecording` on the backend that's triggering session failures at a pretty high rate. Even an hour later, it's still happening.

I'll take a look. This *shouldn't* be happening because our E2E tests are isolated from most processing sessions, but maybe there's a lot of traffic in the Live Tests workspace or something.

**Comment by bvaughn** [comment]
The tests failing on this branch are the exact tests failing on `main`, which is to say, the ones that backend dropped support for due to the age of the runtimes used to record them. So I'm going to land this.

**Pull request merged**